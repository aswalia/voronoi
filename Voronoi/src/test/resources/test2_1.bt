(0.0,3.0) 
(2.0,14.0)
(2.0,28.0) 
(3.0,38.0)
(4.0,0.0) 
(5.0,11.0)
(6.0,10.0) 
(6.0,11.0)
