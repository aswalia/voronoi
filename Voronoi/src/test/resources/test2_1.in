lft:
lft:
lft:
point:(0.0,3.0)
point:(2.0,14.0)
lft:
point:(2.0,28.0)
point:(3.0,38.0)
lft:
lft:
point:(4.0,0.0)
point:(5.0,11.0)
lft:
point:(6.0,10.0)
point:(6.0,11.0)
