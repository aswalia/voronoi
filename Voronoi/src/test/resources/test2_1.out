Vor:
lft and rgt: (3.0,38.0) (6.0,11.0)
lft and rgt: (2.0,28.0) (3.0,38.0)
lft and rgt: (2.0,28.0) (6.0,11.0)
lft and rgt: (2.0,14.0) (2.0,28.0)
lft and rgt: (2.0,14.0) (6.0,11.0)
lft and rgt: (2.0,14.0) (5.0,11.0)
lft and rgt: (0.0,3.0) (2.0,14.0)
lft and rgt: (0.0,3.0) (5.0,11.0)
lft and rgt: (0.0,3.0) (6.0,10.0)
lft and rgt: (0.0,3.0) (4.0,0.0)
lft and rgt: (4.0,0.0) (6.0,10.0)
lft and rgt: (5.0,11.0) (6.0,10.0)
lft and rgt: (6.0,10.0) (6.0,11.0)
lft and rgt: (5.0,11.0) (6.0,11.0)
lft and rgt: (0.0,3.0) (2.0,28.0)
